website/settings/defaults.py

```diff
@@ -317,8 +333,59 @@ def parent_dir(path):
             'schedule': crontab(minute=0, hour=0),
             'args': ('email_digest',),
         },
+        'refresh_box': {
+            'task': 'scripts.refresh_box_tokens',
+            'schedule': crontab(minute=0, hour= 2),  # Daily 2:00 a.m
+            'args': (DAYS, DRY_RUN,),
```

Comment:
once the above default values are set, these args can be simply `dry_run=False`.